<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolutionPage</class>
 <widget class="QWizardPage" name="SolutionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="title">
   <string>Data updating</string>
  </property>
  <property name="subTitle">
   <string>This may take several minutes. Please wait...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="pages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="downloadPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="out">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="generalPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="addLayout" stretch="1,0">
         <item>
          <widget class="QLabel" name="notFoundLbl">
           <property name="text">
            <string>Application &quot;%1&quot; not found.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addBtn">
           <property name="toolTip">
            <string>Add application</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/plus</normaloff>:/icons/plus</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="apps">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="pageLayout" stretch="0,0,0,0,0">
         <item>
          <widget class="QPushButton" name="solutionsBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Solutions</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/solution</normaloff>:/icons/solution</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="lSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="leftBtn"/>
         </item>
         <item>
          <widget class="QLabel" name="page">
           <property name="text">
            <string>1 from 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rightBtn"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
